keps/sig-apps/2185-random-pod-select-on-replicaset-downscale/README.md

```diff
@@ -302,21 +312,20 @@ algorithm falls back to age. (Pod age is the final criteria in the sorting algor
 want to measure this ratio for deletions which don't use this feature, as those may validly fall 
 outside the desired range).
 
-* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
   There should be no values `>2` in the above metric when the Pod Cost annotation is unset 
   (see https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2255-pod-cost) and 
   the pod's deletion was based on a timestamp comparison (rather than, for example, pod state).
 
-* **Are there any missing metrics that would be useful to have to improve observability 
-of this feature?**
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
   Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
   implementation difficulties, etc.).
```

Comment:
This answer looks wrong. The answer can be that we didn't find any other gaps that could be covered by metrics.